src/Footer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Footer = (props) =>{
+ return (
+   <ul className="contact-container">
+     {props.contact.map((lineString) => {
```

Comment:
`lineString` is an odd variable name 🤔  ~Could we think of anything better to use?